assets/src/dashboard/templates/index.js

```diff
@@ -26,16 +27,56 @@ import getFitnessStoryData from './data/fitness';
 import getTravelStoryData from './data/travel';
 import getWellbeingStoryData from './data/wellbeing';
 
+const migrationCache = {};
```

Comment:
Should we use a `Map` here since it's more geared towards quickly looking up data and as a cache?

Reply:
Oh yeah, I can dig it.  I'll update.